server/src/main/java/org/opensearch/index/translog/transfer/TranslogTransferManager.java

```diff
@@ -162,19 +164,15 @@ private void downloadToFS(String fileName, Path location, String primaryTerm) th
     }
 
     public TranslogTransferMetadata readMetadata() throws IOException {
-        return transferService.listAll(remoteMetadataTransferPath)
-            .stream()
-            .max(TranslogTransferMetadata.METADATA_FILENAME_COMPARATOR)
-            .map(filename -> {
-                try (InputStream inputStream = transferService.downloadBlob(remoteMetadataTransferPath, filename);) {
-                    IndexInput indexInput = new ByteArrayIndexInput("metadata file", inputStream.readAllBytes());
-                    return new TranslogTransferMetadata(indexInput);
-                } catch (IOException e) {
-                    logger.error(() -> new ParameterizedMessage("Exception while reading metadata file: {}", filename), e);
-                    return null;
-                }
-            })
-            .orElse(null);
+        return transferService.listAll(remoteMetadataTransferPath).stream().max(METADATA_FILENAME_COMPARATOR).map(filename -> {
+            try (InputStream inputStream = transferService.downloadBlob(remoteMetadataTransferPath, filename);) {
+                IndexInput indexInput = new ByteArrayIndexInput("metadata file", inputStream.readAllBytes());
+                return new TranslogTransferMetadata(indexInput);
+            } catch (IOException e) {
+                logger.error(() -> new ParameterizedMessage("Exception while reading metadata file: {}", filename), e);
+                return null;
+            }
+        }).orElse(null);
```

Comment:
is this unintended change ?

Reply:
Did a static import after which the spotlessApply did this. No code change done really.